### PR TITLE
core(inspector-issues): do not complain about a CSP violation caused by Lighthouse

### LINKED ISSUE #11862: tap-targets causes CSP violation by injecting style sheet
#### Provide the steps to reproduce
1. Run LH 7 on https://almanac.httparchive.org/en/2020/

#### What is the current behavior?

Note that it [gets 93% on Best Practices audit due to Content Security Policy issue](https://www.webpagetest.org/lighthouse.php?test=201220_DiGY_f16b0100aac4fae06afc5f67957f8584&run=3#best-practices):
CSP issue affecting Best Practices Audit

#### What is the expected behavior?

Running Lighthouse in Chrome (v6.4.0) does indeed show a CSP violation but Lighthouse v6.4.0 doesn't complain about this while 7.0.0 does:

CSP violation in console

Digging into this more, the inline style is inserted by Lighthouse itself:

https://github.com/GoogleChrome/lighthouse/blob/2cbbd289be33a586462ed248179df7d7b692ca53/lighthouse-core/gather/gatherers/seo/tap-targets.js#L160-L177

Bit unfair to complain about a problem you are creating 🙂

#### Environment Information
* Affected Channels: CLI (as used by WebPageTest) only for now as 7.0.0 has not rolled out much yet
* Lighthouse version: 7.0.0
* Chrome version: N/A - running Lighthouse outside of Chrome as 7.0.0 not release yet.
* Node.js version: N/A
* Operating System: N/A

**Related issues**

### PR DESCRIPTION
**Pull request by Dimitris-Toulis**
**Summary**

Fixes #11862

**Related Issues/PRs**
#11862

### CODE REVIEW

**Comment by Dimitris-Toulis** [comment]
@googlebot I signed it!

**Comment by connorjclark** [comment]
FYI you can just run `yarn lint --fix`.

File: lighthouse-core/audits/dobetterweb/inspector-issues.js [hunk 1]
```diff
@@ -175,8 +175,16 @@ class IssuesPanelEntries extends Audit {
...
     const cspIssues = issues.contentSecurityPolicy.filter(issue => {
       // kTrustedTypesSinkViolation and kTrustedTypesPolicyViolation aren't currently supported by the Issues panel
-      return issue.contentSecurityPolicyViolationType !== 'kTrustedTypesSinkViolation' &&
-        issue.contentSecurityPolicyViolationType !== 'kTrustedTypesPolicyViolation';
+      return (
+        issue.contentSecurityPolicyViolationType !== 'kTrustedTypesSinkViolation' &&
+        issue.contentSecurityPolicyViolationType !== 'kTrustedTypesPolicyViolation' &&
+        // filter csp issue from tap-targets gatherer
+        !(
+          issue.blockedURL === undefined &&
```
Comment (connorjclark) [inline lighthouse-core/audits/dobetterweb/inspector-issues.js:183]: I didn't confirm that `blockedURL` would only be blank from code that Lighthouse injects. Does that seem to be the case?

**Review by connorjclark — CHANGES_REQUESTED** [review]
Thanks for picking this up!

We'll want an assertion in our smoke tests that this actually filters out our unwanted issue. See `dbw-expectations` and the smoke test README https://github.com/GoogleChrome/lighthouse/blob/psi-elscreens/lighthouse-cli/test/smokehouse/readme.md

File: lighthouse-core/audits/dobetterweb/inspector-issues.js (hunk 1 above)
Comment (Dimitris-Toulis) [reply lighthouse-core/audits/dobetterweb/inspector-issues.js:183]: I got that from: [#11862#issuecomment-788351373](https://github.com/GoogleChrome/lighthouse/issues/11862#issuecomment-788351373)

**Comment by Dimitris-Toulis** [comment]
How to test that without applying a CSP to the dbw_tester.html file which could break other tests?

**Pull request closed without merging**